inc/customizer/extend-custom-controls/src/color-group/color-group-component.js

```diff
@@ -1,4 +1,9 @@
 import PropTypes from 'prop-types';
+const {__} = wp.i18n;
```

Comment:
```suggestion
import { __ } from '@wordpress/i18n';
```

Maybe `@wordpress/i18n` will also need to be added to npm dependencies.

Reply:
Corrected ✔️